src/controllers/PenaltyDetailsController.ts

```diff
@@ -24,12 +24,12 @@ const navigation = {
 };
 
 const sanitizeForm = (body: PenaltyIdentifier) => {
-    const legacyPrefixChars: string = 'PEN';
-    const penalty = body.penaltyReference;
+    const legacyPrefix: string = 'PEN';
+    const penalty = body.penaltyReference.toUpperCase();
```

Comment:
i've noticed we are setting the penalty number to uppercase in sanitizeLegacyPenalty(), do we still need to do this?

Reply:
Can you also add the type please